src/components/grid_overlay/grid_overlay.ts

```diff
@@ -208,3 +222,17 @@ GridOverlay.props = {
   onGridMoved: Function,
   gridOverlayDimensions: String,
 };
+
+const CURSOR_SVG = /*xml*/ `
```

Comment:
This should probably be in icon.ts. And maybe the icon design coule be improved

Reply:
And w ehsould probably put it if a Css variable (if we support them), or in a css class, because having this chunk inlinde fucks up the document inspection

Reply:
You could put the `cursor : ${CURSOR_SVG}` in a `paint-cursor` class, and enable/disable the class with some code. I think that would make the code more readable. At least the svg would be in the CSS and not in the HTML.

Reply:
I agree with Adrien's proposition. Let's use a class